updating/installing-update-service.adoc

```diff
@@ -22,7 +22,32 @@ include::modules/update-service-overview.adoc[leveloffset=+1]
 == Prerequisites
 
 * For more information on installing Operators, see xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
-* If the release images are contained in a secure registry, complete the steps in xref:../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access].
+
+[id="update-service-registry-configuration"]
+=== Configuring additional trust stores for image registry access
```

Comment:
The proposed title _has to_ change because it is identical to the existing topic title for the link in the foll para.  I'll hazard a complete guess--I don't know the content area:

"Configuring access to a secured update service registry"

No idea if that works or why it might be terrible.

Reply:
Changed it to 
```
[id="registry-configuration-for-update-service"]
=== Configuring access to a secured registry for the OpenShift update service
```
Does this work for you?